deliverables/README.md

```diff
@@ -2,13 +2,17 @@
 #### Required
 ##### Questions
 - What is a framework?
+ Like many web frameworks, Ruby on Rails uses the model–view–controller (MVC) pattern to organize application programming. In a  default configuration, a model in the Ruby on Rails framework maps to a table in a database, and a Ruby file.
```

Comment:
Please link to source when you use verbatim text:  http://en.wikipedia.org/wiki/Ruby_on_Rails